Makefile

```diff
@@ -79,11 +81,15 @@ install:
 testbuild: TESTS := $(shell $(GO) list -tags '$(TAGS)' $(PKG))
 testbuild: GOFLAGS += -c
 testbuild:
+	ROOT="$(DIR)"; \
 	for p in $(TESTS); do \
```

Comment:
I know, this is why we need the loops, otherwise we have to do a loop outside the makefile. It makes more sense to have the default make rule build everything